course_access_groups/serializers.py

```diff
@@ -1,13 +1,75 @@
-from rest_framework import serializers
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, unicode_literals
 
+from rest_framework import serializers
+from openedx.core.lib.api.serializers import CourseKeyField
 from course_access_groups.models import (
-    CourseAccessGroup,
+    Group,
+    CourseGroup,
+    Membership,
+    Rule,
 )
 
 
 class GroupSerializer(serializers.ModelSerializer):
+    id = serializers.ReadOnlyField()
+    organization_name = serializers.ReadOnlyField(source='organization.name')
+
+    class Meta:
+        model = Group
+        fields = [
+            'id', 'name', 'description', 'organization', 'organization_name',
+        ]
+
+
+class MembershipSerializer(serializers.ModelSerializer):
+    id = serializers.ReadOnlyField()
+    user_email = serializers.ReadOnlyField(source='user.email')
+    user_username = serializers.ReadOnlyField(source='user.username')
```

Comment:
Isn't 'user username' redundant? Why not just 'username'?

Reply:
It's a convention that I'm used to do for denoting that this attribute is from another related entity. What do you think?

Reply:
It's a little weird to me, but I won't protest too much. Keep it

Reply:
Thanks. I'll probably regret it in the future, but for now I can't see what's wrong with it.